backend/infrahub/core/schema/definitions/core.py

```diff
@@ -797,6 +798,12 @@
                     "default_value": AccountRole.READ_ONLY.value,
                     "enum": AccountRole.available_types(),
                 },
+                {
+                    "name": "status",
+                    "kind": "Text",
+                    "default_value": AccountStatus.ACTIVE.value,
+                    "enum": AccountStatus.available_types(),
```

Comment:
I think it would be good to use a dropdown instead in order to have labels & colors